rush.json

```diff
@@ -767,6 +767,12 @@
       "reviewCategory": "libraries",
       "shouldPublish": true
     },
+    {
+      "packageName": "@rushstack/heft-workbox-plugin",
```

Comment:
It'd be useful to have a test project.

Reply:
Agreed. Should this be several different test projects for different classes of application? I have a small test project I was using to test out the workbox, esbuild and browsersync plugins together. I can bring that in as another PR if the team wants?

Reply:
Sounds useful to me.